lib/render/set-claim.php

```diff
@@ -36,3 +37,40 @@ function renderNewClaimsComponent(int $count): void
     echo "<div class='morebutton'><a href='/claimlist.php'>more...</a></div>";
     echo "</div>";
 }
+
+/**
+ * Creates the Completed Set Claims component.
+ */
+function renderFinishedClaimsComponent(int $count): void
+{
+    echo "<div class='component'>";
+    echo "<h3>Finished Set Claims</h3>";
```

Comment:
"New Sets"?

Reply:
It's also completed revisions, and it complements the "new set claims" that's further down the page.

Reply:
Was about to write that into a discussion - probably still should. Maybe that's all just a matter of taste.

I find the increasing use of "claims" on the front page problematic.

Claims - while unfortunately necessary - just describe the underlying processes that we had to introduce due to the shortcomings of what the site can do.

Personally, I'd rather focus on what this should become - a more collaborative system. "Claiming" something, to me, implies ownership and that should not have to exist.
Coming to the site as a player "claims" requires more explanation than calling it "new/revised sets" and/or "sets in progress".

Anyways, this probably not the best place to discuss this but wanted to have written down somewhere.

Reply:
Would something like `Recently Finished Sets / Revisions` work better in your opinion?

Reply:
"Recently Finished" seems overly wordy. Just "New Sets/Revisions" seems fine.

I'm not sure about better wording for "New Set Claims" though. "Sets in Progress" (as suggested above) seems like the best candidate. Then you'd also want to change "Claimed On" to "Started".

Reply:
> Was about to write that into a discussion - probably still should.

This probably deserves a discussion and a separate PR as the "claim" nomenclature permeates the site.

At a minimum, the menu item "Community > Claim List" should also be updated to "Sets in Progress". The rest of the references are arguably acceptable (even the "Claim List" page itself) as it does represent a view of the process.

Reply:
I'd see "recently" as implied/superfluous, too. It's the front page so it should be the new stuff only anyways.

Agreed, from a developer's perspective it's all about claims; I'd keep public pages (except for where it's explicitly about claims) targeted at players and go for "sets" as that's what a player interacts with eventually, not underlying claim.